CMakeLists.txt

```diff
@@ -155,12 +155,20 @@ endif()
 
 install(TARGETS
     graphqlservice
-    graphqljson
   EXPORT cppgraphqlgen-targets
   RUNTIME DESTINATION bin
   ARCHIVE DESTINATION lib
   LIBRARY DESTINATION lib)
 
+if(USE_RAPIDJSON)
```

Comment:
Just turning off `USE_RAPIDJSON` fails to build the tests. I want to see if I can make that conditional on providing an implementation of `graphqljson` instead, either with `USE_RAPIDJSON` or by including another implementation.